### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by John-Colvin**
test the dub build as well.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+ - ldc-0.15.1
+ - gdc
+
+addons:
+  apt:
+    packages:
+      liblapack-dev
+
+script:
+ - make all
```
Comment (9il) [inline .travis.yml]: Dose we need make support?

Comment (John-Colvin) [inline .travis.yml]: Seeing as it's there, we should test it. Whether or not make is necessary in the long run, that's a different discussion.

**Comment by kyllingstad** [comment]
Why specify the compiler versions? Isn't it better to always test with the latest ones?

File: .travis.yml (hunk 1 above)
Comment (kyllingstad) [inline .travis.yml]: make support is there for the benefit of D-APT.

**Comment by John-Colvin** [comment]
2 advantages:
* explicitly testing/supporting the latest version, in case travis is out-of-date.
* test compatibility with older compilers, not everyone has the latest version

**Comment by 9il** [comment]
> test compatibility with older compilers, not everyone has the latest version

The problem is not to download new version, sometimes people have not time to keep their project up-to-date with latest compiler version.

**Comment by John-Colvin** [comment]
I'm not a makefile guy really. Is calling ```make command.make``` followed by ```make all``` the right way to do things?

**Comment by John-Colvin** [comment]
I've dropped the makefile test for now. The makefile seems to be broken with gdc in a few different ways

**Comment by kyllingstad** [comment]
> Is calling `make command.make` followed by `make all` the right way to do things?

No, `make` (without "all") should be sufficient. This will build the `all` target, as it's the one listed first in the makefile. `command.make` gets included by `Makefile`.

**Comment by kyllingstad** [comment]
> I've dropped the makefile test for now.

I think that is OK. It's mainly there for Linux packagers, and if they have trouble with it, they can fix it too. :-)

**Pull request merged**